### PR TITLE
Human-Assistant Formatting Updates + Version Updates

### PR DESCRIPTION
**Pull request by zack-anthropic**
- Update v1 to v2
- Use '\n\nHuman:' and '\n\nAssistant:' everywhere
- Use XML tags to separate instructions from data [this is a best practice]
- Various typo cleanup

### CODE REVIEW

File: 01_Generation/00_generate_w_bedrock.ipynb [hunk 1]
```diff
@@ -181,8 +181,8 @@
     "- `amazon.titan-tg1-large`\n",
     "- `ai21.j2-grande-instruct`\n",
     "- `ai21.j2-jumbo-instruct`\n",
-    "- `anthropic.claude-instant-v1`\n",
-    "- `anthropic.claude-v1`"
+    "- `anthropic.claude-instant-v2`\n",
```
Comment (ihmaws) [inline 01_Generation/00_generate_w_bedrock.ipynb:184]: Is there a v2 available for the instant model? I don't see it on Anthropic's main site, let alone on Bedrock.

Please confirm.

File: 04_Chatbot/00_Chatbot_Claude.ipynb [hunk 2]
```diff
@@ -156,7 +156,7 @@
     "from langchain.memory import ConversationBufferMemory\n",
     "\n",
     "cl_llm = Bedrock(\n",
-    "    model_id=\"anthropic.claude-v1\",\n",
+    "    model_id=\"anthropic.claude-v2\",\n",
```
Comment (ihmaws) [inline 04_Chatbot/00_Chatbot_Claude.ipynb:159]: Do you think we should use the instant model here and a few other places a bit more?

I know in this PR you are just updating v1 to v2, but with the price/performance trade-offs of the instant, I wonder if chat and question answering might be good candidates to move to the instant model.

Let me know what you think.

**Review by ihmaws — COMMENTED** [review]
Hi @zack-anthropic, thanks for the PR!

I'll try to walk through the notebook changes in my IDE later today but just a few quick comments for now.

**Review by mani-aiml — APPROVED** [review]

File: 01_Generation/00_generate_w_bedrock.ipynb (hunk 1 above)
Comment (zack-anthropic) [reply 01_Generation/00_generate_w_bedrock.ipynb:184]: Thanks. I'm getting the exact model names from a teammate now.

Comment (zack-anthropic) [reply 01_Generation/00_generate_w_bedrock.ipynb:184]: Will do a find/replace once I've determined what they are.

File: 04_Chatbot/00_Chatbot_Claude.ipynb (hunk 2 above)
Comment (zack-anthropic) [reply 04_Chatbot/00_Chatbot_Claude.ipynb:159]: I think it's better for people to start with the bigger/smarter model to see if they can hit their performance bar, as that's often the first blocker. Then they can optimize further from there.

**Pull request merged**